biosppy/features/frequency.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+from .. import utils
+from . import time
+from scipy import interpolate
+
+
+def getbands(frequencies, fband= [0, 1]):
```

Comment:
Here I would suggest removing the default parameter value of `fband` since all calls of this function specify the fband value.

```suggestion
def getbands(frequencies, fband):
```